package.json

```diff
@@ -17,23 +17,22 @@
   },
   "homepage": "https://github.com/elierotenberg/react-nexus",
   "devDependencies": {
+    "babel-eslint": "^3.0.1",
     "del": "^1.1.1",
+    "eslint-plugin-react": "^2.2.0",
     "gulp": "^3.8.11",
-    "gulp-babel": "^4.0.0",
+    "gulp-babel": "^5.1.0",
+    "gulp-eslint": "^0.11.1",
     "gulp-insert": "^0.4.0",
-    "gulp-jshint": "^1.9.2",
-    "gulp-plumber": "^0.6.6",
-    "gulp-rename": "^1.2.0",
-    "gulp-sourcemaps": "^1.3.0",
-    "gulp-util": "^3.0.3",
-    "jshint-stylish": "^1.0.0"
+    "gulp-plumber": "^1.0.0",
+    "gulp-util": "^3.0.4"
   },
   "dependencies": {
-    "babel": "^4.3.0",
-    "bluebird": "^2.9.10",
-    "lodash": "^3.2.0",
-    "nexus-flux": "^2.0.1",
+    "babel": "^5.1.13",
+    "bluebird": "^2.9.24",
+    "lodash": "^3.7.0",
+    "nexus-flux": "^3.2.0",
```

Comment:
I guess this version is to be published sooner or later. It is not available right now when I try to `npm install nexus-flux@3.3.0`